### DISCUSSION TITLE
Support of Visual Studio Code for the Web

### DISCUSSION THREAD

**Discussion by kcleung** [bernhard-42/vscode-ocp-cad-viewer#140] [Ideas]
Currently, the OCP CAD viewer extension does not support Visual Studio Code for the Web, and can only be run from a local instance of Visual Studio Code

However, would it be technically possible to implement the extension for Visual Studio Code for the Web?

**Comment by bernhard-42** [comment]
https://code.visualstudio.com/docs/editor/vscode-web#_relationship-to-vs-code-desktop
> VS Code for the Web provides a browser-based experience for navigating files and repositories and committing lightweight code changes. However, if you need access to a runtime to run, build, or debug your code, you want to use platform features such as a terminal, or you want to run extensions that aren't supported in the web, we recommend moving your work to the desktop application

OCP CAD Viewer needs the terminal for running the backend and no idea whether C++ based Python modules would work.
In addition, it would be yet another platform to build for and test, so I am not planning to support it.
